src/main/java/com/intuit/graphql/orchestrator/batch/EntityFetcherBatchLoader.java

```diff
@@ -142,13 +157,41 @@ private Map<String, Object> createRepresentation(
         return entityRepresentation;
     }
 
-    private List<Selection> pruneExternalResolvers(String parentTypeName, Field field) {
+    private List<Selection> pruneExternalResolvers(TypeDefinition parentTypeDefinition, Field field) {
         return field.getSelectionSet().getSelections().stream()
         .filter(selection -> {
             Field selectionField = (Field) selection;
-            FieldCoordinates fieldCoordinates = FieldCoordinates.coordinates(parentTypeName, selectionField.getName());
+            FieldCoordinates fieldCoordinates = FieldCoordinates.coordinates(parentTypeDefinition.getName(), selectionField.getName());
 
             return entityServiceMetadata.getFieldResolverContext(fieldCoordinates) == null;
-        }).collect(Collectors.toList());
+        })
+        .map(selection -> {
+            if(!selection.getChildren().isEmpty()) {
```

Comment:
I'm assuming this is to check if selection has sub selections.  Not sure if getChildren() is the way to go.